src/widget.lua

```diff
@@ -518,10 +518,15 @@ w.StaticText = StaticText
 function StaticText:init(text, args)
     Text.init(self, args or {})
 
+    local line_count = 0
```

Comment:
You can skip this by using `#self._lines` below.

Reply:
As I'm new to Lua, I think I read something about tables and misunderstood - maybe it's that you can't loop over them in the same way like you can in Python.... anyway, glad to have learned - fixed (and thanks)!